src/methods/index.test.ts

```diff
@@ -108,15 +109,15 @@ describe("Methods for handling signatory request", () => {
 
   it("should sign data", async () => {
```

Comment:
The inclusion of eserialize is a nice change, and the fix for the test is correct . In this case the handlers.sign example is incorrect and shoudl be hexed as you've applied . Thanks for this update. Please see the general comment about hexToString.